src/osd/OSD.h

```diff
@@ -1927,6 +1927,7 @@ class OSD : public Dispatcher,
   void _preboot(epoch_t oldest, epoch_t newest);
   void _send_boot();
   void _collect_metadata(map<string,string> *pmeta);
+	std::string _collect_compression_algorithms();
```

Comment:
wrong indent.

Reply:
It's to show that this function depends on the other function.

Reply:
i'd suggest follow our coding style before we enforce yours.